src/content/docs/tutorial-create-alerts/create-an-alert.mdx

```diff
@@ -13,37 +17,148 @@ import alertsThresholds from 'images/alerts_screenshot-crop_set-thresholds.webp'
 
 import alertsDetails from 'images/alerts_screenshot-crop_condition-details.webp'
 
-Starting an alert from a chart is one of the easiest ways to set up a New Relic alert condition. Instead of creating a NRQL query from scratch, charts let you use pre-existing NRQL conditions to begin the process. In this example, we'll set up an alert condition using an error response chart.
+import alertsNRQL from 'images/alerts_screenshot-crop_NRQL-query.webp'
 
-<img
-    title="Create chart alert"
-    alt="A screenshot displaying where to create an alert condition from a chart in New Relic"
-    src={alertsCreate}
-/>
+import alertsNewCreate from 'images/alerts_screenshot-crop_create-new-query.webp'
+
+import alertsGoldenCreate from 'images/alerts_screenshot-crop_golden-metrics.webp'
+
+import alertsGoldenEntities from 'images/alerts_screenshot-crop_golden-metric-entities.webp'
+
+You can create an alert condition in New Relic using 1 of 3 options: using a chart, using a NRQL query, or using golden metrics. While the beginning of the creation process is different depending on which option you choose, all the subsequent steps are the same. This means that once you've created a condition one way, the experience you gain from doing so will transfer over to others that you create using either of the other options.
+
+For this tutorial, we've included steps for all 3 of the aforementioned options, so you can get started with your first condition any way you want to. Just select the path you want to follow in the first step, and by the end of the procedure you'll have successfully made your first alert condition using New Relic!
 
 <Callout variant="tip">
 If you need more information on any of the options or terms covered in the steps below, check out our [alerts docs](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/).
     </Callout>
 
 <Steps>
     <Step>
-## Select a chart and define your signal
-
-<img
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Create using a chart
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Create using the guided mode
+        </TabsBarItem>
+        <TabsBarItem id="3">
+            Create using a NRQL query
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="1">
+<SideBySide>
+    <Side>
+Starting from a chart is one of the easiest ways to set up a New Relic alert condition. Instead of creating a NRQL query from scratch, charts let you use pre-existing NRQL queries to begin the process. 
+    
+In this example, we'll set up an alert condition using an error response chart. Follow the steps below with any type of chart you like to begin.
+    </Side>
+    <Side>
+    <img
+    title="Create chart alert"
+    alt="A screenshot displaying where to create an alert condition from a chart in New Relic"
```

Comment:
the way these images are stacked sets up the user to think that the right screenshot (clicking the `...` to get the expanded menu) corresponds with the left text, but the first screenshot goes with step 1 of the succeeding paragraph. I think this would be clearer if you described the steps then put the text in, and maybe removed the side by side.

Reply:
Alternatively, moving step 1 of the stepped instructions to be step one in the first part of the component. something to that effect

Reply:
Hmm, good point. I thought about how best to fix it, I think that moving step 1 up to the left side is the simplest solution and also the most direct in terms of keeping the information to it's implied paired screenshot. It also fills out the white space nicely. See what you think.